### PR TITLE
Config in supertype

### PR DESCRIPTION
**Pull request by sshankar19**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8.10.0
+      - image: cimg/node:12.20.2
```
Comment (jacoblb64) [inline .circleci/config.yml:5]: should we update the "engines" blocks in the various components to be >=12.0 now?

File: components/amorphic-contracts/index.d.ts [hunk 2]
```diff
@@ -1,13 +1,21 @@
 /**
  * application configuration store object
+ *
+ * get => Gets properties from this config object
```
Comment (nsaenz7) [inline components/amorphic-contracts/index.d.ts:4]: 👍 adding comments

**Review by nsaenz7 — DISMISSED** [review]
looks good! moving tests to TS, nice

**Review by jacoblb64 — DISMISSED** [review]

**Review by jacoblb64 — APPROVED** [review]
Looks good, thanks for addressing!

**Review by nsaenz7 — APPROVED** [review]

**Pull request merged**